sklearn/svm/src/libsvm/svm.cpp

```diff
@@ -3101,6 +3101,44 @@ const char *PREFIX(check_parameter)(const PREFIX(problem) *prob, const svm_param
 		free(count);
 	}
 
+	if(svm_type == C_SVC ||
+	   svm_type == EPSILON_SVR ||
+	   svm_type == NU_SVR ||
+	   svm_type == ONE_CLASS)
+	{
+	    PREFIX(problem) newprob;
+          // filter samples with negative weights like algorithm PREFIX(train)
```

Comment:
This style of outdenting the comments is not conventional in this file or repository.

Not sure what you mean by "like algorithm PREFIX(train)".

Reply:
fixed identation and comment